### PR TITLE
Included PR check for minikube

### PR DESCRIPTION
**Pull request by chandrams**
Signed-off-by: Chandrakala Subramanyam <csubrama@redhat.com>

### CODE REVIEW

File: tests/scripts/common/common_functions.sh [hunk 1]
```diff
@@ -457,7 +463,7 @@ function check_server_status() {
 	echo "Server - $SERVER_IP PORT - $PORT"
 
 	#if service does not start within 5 minutes (300s) fail the test
-	timeout 30 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://${SERVER_IP}:${PORT})" != "200" ]]; do sleep 1; done' || false
+	timeout 300 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://${SERVER_IP}:${PORT})" != "200" ]]; do sleep 1; done' || false
```
Comment (dinogun) [inline tests/scripts/common/common_functions.sh:466]: Five mins seems excessive as the job on PR check will be running that long. Why is this needed?

Comment (chandrams) [reply tests/scripts/common/common_functions.sh:466]: This timeout is to check if the HPO service has started or not. If it is not started it will wait for 5 mins before timing out. We can reduce it to 2 mins. Atleast on the github hosted runners, I have seen services taking time to start.

Comment (dinogun) [reply tests/scripts/common/common_functions.sh:466]: Yes, please reduce it to 2 mins, that sounds good

**Review by dinogun — APPROVED** [review]
LGTM
